examples/Cargo.toml

```diff
@@ -14,7 +14,18 @@ anyhow = "1.0.32"
 serde_cbor = "0.11"
 humantime = "2.0.1"
 rand = "0.8.3"
+dtn7 = { path = "../core/dtn7" }
```

Comment:
should have path as well as version appended to avoid problems with `cargo publish`

Reply:
Version added.